modules/WebPublishPlugin/src/main/resources/net/clementlevallois/web/publish/plugin/controller/Bundle.properties

```diff
@@ -0,0 +1,47 @@
+OpenIDE-Module-Long-Description=This plugin converts your network into a gexf file. This file is sent to a "Gist" of Github, where it becomes publicly accessible. Then, a link is created, which allows you to view, explore and share your network with the Retina web application. Be careful, do not publish your networks if the data is confidential\!
+general.message.warning_confidentiality=\ud83d\udd25Warning\! Do not publish your network if it contains confidential data\!
+general.message.error_message=Error message\:
+general.message.warning.network_too_big=Warning. The network has either more than 10,000 entities or more than 20,000 links. It will probably make the web version impractical to use. Do you really want to publish the network?
+general.message.success_switch_to_publish=Hit. You can go to "publish".
+general.message.url_published_gexf=Your network URL is\:
+JPanelWebExport.jTextFieldWebsiteLoginUrl.text=https\://github.com/login/device
+JPanelWebExport.step3.info2=On this website, click on "Install"
+general.message.info.url_will_appear_below=\ud83d\udd17The link (url) of your network will appear below
+JPanelWebExport.step3.info1=To do this, go to the website\:
+general.message.file_sent_from_gephi=File sent from Gephi
+JPanelWebExport.jTextField2.text=https\://github.com/apps/gephi-lite
+general.message.then_wait_for_code=then wait for the code and copy it
+JPanelWebExport.jButtonConnectToGephiLite.text=Connect to Gephi-Lite
+general.message.error.no_token=Unable to publish. Did you do the settings correctly?
+JPanelWebExport.step3.title=2. Confirm the installation of the gephi-lite application
+general.message.success_ready_to_publish=Everything should be fine now\! Go to the "publish" tab of this window and publish your network\!
+general.message.error.cant_retrieve_user_code=Unable to retrieve their user code. Are you properly connected to Github?
+general.message.error.no_open_project=You must create or open a project otherwise there is nothing to publish.
+JPanelWebExport.step6.info1=Then click on "continue" then "authorize"
+OpenIDE-Module-Name=Publish your network to the web
+general.message.errors_appear.here=any error messages will appear here
+JPanelWebExport.tabGithub.TabConstraints.tabTitle=Set up (to do just once)
+general.message.error.gexf_not_retrieved=Unable to create gexf file from your network.
+general.message.success_reset=The reset was successful.
+general.message.error.gist_creation=Error creating online gexf file. Error code\:
+OpenIDE-Module-Short-Description=This plugin allows you to publish, visualize and share your network on the web, directly from Gephi.
+JPanelWebExport.step6.title=4. Paste the code on this website\:
+general.message.error.empty_network=Your network is empty. Create a network to publish it.
+JPanelWebExport.step2.title=2. Login to your Github Account
+JPanelWebExport.step1.title=1. You need a Github account and be logged in
+general.message.plugin.name=Publish to the web
```

Comment:
As a convention, all actions menu that open a new menu/window should end with three dots "..."

Reply:
✅